### ISSUE TITLE
Asking for help, I encountered a bug

### ISSUE THREAD

**Issue by fsted** [Atlas200dk/sample-imageinpainting-HiFill#22]
InvalidArgumentError (see above for traceback): logits must be 2-dimensional
	 [[Node: Softmax = Softmax[T=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"](mul_att)]]

Who can tell me how to do 。。。

**Comment by watertianyi** [comment]
用 tf=1.13.1 解决了它
